src/midi/PlayerThread.cpp

```diff
@@ -52,21 +52,20 @@ void PlayerThread::setInterval(int i){
 
 void PlayerThread::run(){
 
-	if(timer){
```

Comment:
`timer.start()` resets the existing timer. Why are we recreating the timer every 15 ms?